docs/user-guide/cli-using-understanding-core-command-groups.md

```diff
@@ -1,205 +1,273 @@
 
-The zos-tso command group lets you issue TSO commands and interact with TSO address spaces on z/OS systems.
+The `zos-tso` command group lets you issue TSO commands and interact with TSO address spaces on z/OS systems.
 
-With the zos-tso command group, you can perform the following tasks:
+With the `zos-tso` command group, you can perform the following tasks:
 
 * Execute REXX scripts
 * Create a TSO address space and issue TSO commands to the address space.
 * Review TSO command response data in Zowe CLI.
 
-**Note:** For more information about `zos-tso` syntax, actions, and options, open Zowe CLI and issue the following
+:::note
+
+For more information about `zos-tso` syntax, actions, and options, open Zowe CLI and issue the following
 command:
 
 ```
 zowe zos-tso -h
 ```
 
+:::
+
+## zos-workflows
+
+The `zos-workflows` command group lets you create and manage z/OSMF workflows on a z/OS system.
+
+With the `zos-workflows` command group, you can perform the following tasks:
+
+* Create or register a z/OSMF workflow based on the properties on a z/OS system.
+* Start a z/OSMF workflow on a z/OS system.
+* Delete or remove a z/OSMF workflow from a z/OS system.
+* List the z/OSMF workflows for a system or sysplex.
+
+:::note
+
+For more information about `zos-workflows` syntax, actions, and options, open Zowe CLI and issue the following command:
+
+```
+zowe zos-workflows -h
+```
+
+:::
+
 ## zosmf
 
-The zosmf command group lets you work with Zowe CLI profiles and get general information about z/OSMF.
+The `zosmf` command group lets you work with Zowe CLI profiles and get general information about z/OSMF.
 
-With the zosmf command group, you can perform the following tasks:
+With the `zosmf` command group, you can perform the following tasks:
 
-* Create and manage your Zowe CLI `zosmf` profiles. Profiles let you store configuration information for use on multiple commands. You can create a profile that contains your username, password, and connection details for a particular mainframe system, then reuse that profile to avoid typing it again on every command. You can switch between profiles to quickly target different mainframe subsystems. For more information, see [Team configurations](../user-guide/cli-using-using-team-profiles.md).
-* Verify that your profiles are set up correctly to communicate with z/OSMF on your system. For more information, see [Test Connections to z/OSMF](../user-guide/cli-using-test-zosmf-connection).
+* Create and manage your Zowe CLI `zosmf` profiles. Profiles let you store configuration information for use on multiple commands. You can create a profile that contains your username, password, and connection details for a particular mainframe system, then reuse that profile to avoid typing connection information again on every command. You can switch between profiles to quickly target different mainframe subsystems. For more information, see [Team configurations](../user-guide/cli-using-using-team-profiles.md).
```

Comment:
You cannot create or manage zosmf profiles from the zosmf command group. I would delete this bullet point.